app/controllers/trashed_issues_controller.rb

```diff
@@ -14,14 +14,19 @@ class TrashedIssuesController < ApplicationController
   def show
     @issue = @trashed.rebuild
     @relations = @issue.relations
-    @journals = @issue.journals
-    @journals.each.with_index(1) do |journal, indice|
-      journal.indice = indice
-    end
+    @journals = sorted_journals_with_index(@issue.journals)
```

Comment:
メソッドを追加しなくても
```suggestion
    @journals = @issue.journals.sort_by(&:created_on)
    @journals.each.with_index(1) do |journal, indice|
      journal.indice = indice
    end
```
だけでいいのかなと思いました

Reply:
その通りですね✨
修正しました